app/controllers/util/FormField.scala

```diff
@@ -115,24 +118,22 @@ case class DateField(
 }
 
 object FormField {
-
-  val dropdownOptionNotSelectedError = "There was no value selected for the %s."
+  val dropdownOptionNotSelectedError = "Search for and select at least one %s"
   val invalidDropdownOptionSelectedError = "Option '%s' was not an option provided to the user."
-  val emptyValueError = "There was no %s entered for the %s."
-  val numberError = "%s entered must be a whole number."
-  val negativeNumberError = "%s must not be a negative number."
-  val invalidYearError = "The year should be 4 digits in length."
-  val invalidDateError = "%s is an invalid %s number."
-  val invalidDayError = "%s does not have %d days."
-  val invalidDayForLeapYearError = "%s %d does not have %d days in it."
-  val futureDateError = "%s date cannot be a future date."
-  val radioOptionNotSelectedError = "There was no value selected for %s."
+  val emptyValueError = "The %s must contain a %s"
+  val wholeNumberError = "The %s must be a whole number, like %s"
+  val wholeNumberError2 = "The %s of the %s must be a whole number, like %s"
+  val negativeNumberError = "The %s cannot be a negative number"
+  val invalidYearError = "The year of the %s must contain 4 digits"
+  val futureDateError = "The date of the %s must be in the past"
+  val invalidDateError = "The %s of the %s must be between 1 and %s"
+  val invalidDayError = "%s does not have %d days in it. Enter the day for the %s between 1 and %d"
+  val radioOptionNotSelectedError = "Select if the %s is sensitive to the public"
   val invalidRadioOptionSelectedError = "Option '%s' was not an option provided to the user."
   val tooLongInputError = "%s must be %s characters or less"
-
-  def inputModeToFieldType(inputMode: String): String = {
-    if (inputMode.equals("numeric")) "number" else "text"
-  }
+  val closurePeriodNotEnteredError = "Enter the number of years the record is closed from the closure start date"
+  val dependencyNotEntered = "Add an %s for this record"
```

Comment:
Will this always be `Add an` ? It works for field names beginning with a vowel but it sounds odd otherwise, e.g `Add an closure period for this record`

Reply:
At the moment only 2 fields have `Add an`. Closure period has it's own error message. Example below
![Screenshot from 2023-02-10 10-36-23](https://user-images.githubusercontent.com/9700622/218073576-6d6f6459-2bbb-4aaf-90d7-e3005704b0e5.png)

Reply:
I was more thinking for the future, if we update the database with new fields that need this. The rules are quite complicated around it so maybe we'll end up doing `Add a(n) xxx` or something. Guess we can leave it for now.

Reply:
oh right I know what you mean. Julius mentioned something about having a separate table for errors but I guess we can think about it when we start to implement more fields.